data/MI/incentives.json

```diff
@@ -465,73 +462,15 @@
     "program": "mi_dTE_airConditioners",
     "amount": {
       "type": "dollar_amount",
-      "number": 850
-    },
-    "owner_status": [
-      "homeowner"
-    ],
-    "short_description": {
-      "en": "$850 for 19+ SEER or 18+ SEER2 Air Source Heat Pump."
-    }
-  },
-  {
-    "id": "MI-22",
-    "authority_type": "utility",
-    "authority": "mi-dte",
-    "payment_methods": [
-      "rebate"
-    ],
-    "item": "heat_pump_air_conditioner_heater",
-    "program": "mi_dTE_airConditioners",
-    "amount": {
-      "type": "dollar_amount",
-      "number": 750
+      "number": 850,
```

Comment:
combined MI-22, 23, 24 into this incentive and made it a range. they're all heat pump rebates for various efficiencies